sharepoint/sharepoint-ps/sharepoint-online/Copy-SPOPersonalSitePage.md

```diff
@@ -0,0 +1,186 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/copy-spopersonalsitepage
+applicable: SharePoint Online
+title: Copy-SPOPersonalSitePage
+schema: 2.0.0
+author: xuyangzou
+ms.author: xuyangzou, spodeanu
+ms.reviewer:
+---
+
+# Copy-SPOPersonalSitePage
+
+## SYNOPSIS
+
+**What does this command do?**
+
+This cmdlet command allows you to relocate existing SharePoint pages by utilizing an existing copy operation. We will also copy any assets associated with the SharePoint pages to the new destination. We offer two methods for relocating pages:
+- Copy: This method keeps the original page intact while creating a duplicate at the new location.
+- Move: This method creates a new copy at the new location and deletes the original page from the source.
+
+**Where can I move the existing page(s) to and from?**
+
+| Source | Destination |
+| :------------------- | :---------- | 
+| SharePoint pages library in OneDrive  | SharePoint pages library in OneDrive | 
```

Comment:
>OneDrive [](http://example.com/codeflow?start=30&length=8)

(here and elsewhere)

should this be explicit as "OneDrive for Business" instead of just "OneDrive"?